### PR TITLE
RAIL-2777 Proofread the changelog

### PR DESCRIPTION
**Pull request by evgeniabogranskaya**
---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: common/changes/@gooddata/sdk-ui-all/lhq-sd-1179_2020-10-23-10-23.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Load date format for InsightView component",
+            "comment": "The load date format is added to the InsightView component.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/lhq-sd-1179_2020-10-23-10-23.json:5]: This is not correct, the idea here is that the dateFormat that the user/workspace has defined is now loaded in the InsightView and used in the visualizations. So the load part is important there.

File: common/changes/@gooddata/sdk-ui-all/RAIL-2317_2020-10-20-10-06.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Added hooks alternative for Execute component - useCancelablePromise, useDataView, useExecution and useDataExport",
+            "comment": "Alternatives to hooks (useCancelablePromise, useDataView, useExecution, and useDataExport) are added to the Execute component.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/RAIL-2317_2020-10-20-10-06.json:5]: This is inverted: it should be something like
> Hook alternatives for Execute component are added.

In other words, we did not have hooks and now we do.

File: common/changes/@gooddata/sdk-ui-all/anvuong-sd1107_2020-10-08-11-57.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Passes all fetched display forms of each attribute in an insight to AD in an InsightReferencesQuery",
+            "comment": "All displayForms fetched from an attribute in an insight are passed to Analytical Designer in an InsightReferencesQuery.",
```
Comment (no23reason) [inline common/changes/@gooddata/sdk-ui-all/anvuong-sd1107_2020-10-08-11-57.json:5]: I will probably remove this entry in another PR, it makes little sense for end users, sorry about that 😞

Comment (evgeniabogranskaya) [reply common/changes/@gooddata/sdk-ui-all/anvuong-sd1107_2020-10-08-11-57.json:5]: Let me remove this!!!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1820/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3430/ : SUCCESS in 10m 33s

File: common/changes/@gooddata/sdk-ui-all/lhq-sd-1179_2020-10-23-10-23.json (hunk 1 above)
Comment (evgeniabogranskaya) [reply common/changes/@gooddata/sdk-ui-all/lhq-sd-1179_2020-10-23-10-23.json:5]: Ahhhh okay! Rewriting...

File: common/changes/@gooddata/sdk-ui-all/RAIL-2317_2020-10-20-10-06.json (hunk 2 above)
Comment (evgeniabogranskaya) [reply common/changes/@gooddata/sdk-ui-all/RAIL-2317_2020-10-20-10-06.json:5]: Gotcha, thanks!

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/584/ : SUCCESS in 29s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/845/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3435/ : SUCCESS in 18m 18s

**Pull request merged**